include/picongpu/particles/densityProfiles/FreeFormulaImpl.hpp

```diff
@@ -21,40 +21,43 @@
 
 #include "picongpu/simulation_defines.hpp"
 #include "picongpu/simulation/control/MovingWindow.hpp"
+#include "picongpu/particles/functor/User.hpp"
 
 
 namespace picongpu
 {
 namespace densityProfiles
 {
-
-template<typename T_ParamClass>
-struct FreeFormulaImpl : public T_ParamClass
-{
-    using ParamClass = T_ParamClass;
-
-    template<typename T_SpeciesType>
-    struct apply
+    template< typename T_UserFuntor >
+    struct FreeFormulaImpl : public particles::functor::User< T_UserFuntor >
```

Comment:
Typo: should be `T_UserFunctor`, not `T_UserFuntor` .

Can we also at least briefly list the requirements for this class?